package.json

```diff
@@ -47,7 +49,7 @@
     "regenerator-runtime": "^0.13.7",
     "text-encoding": "^0.7.0",
     "typescript": "^4.3.5",
-    "vite": "^2.6.14",
+    "vite": "^2.9.8",
```

Comment:
Updating Vite to include fix from https://github.com/vitejs/vite/pull/7098 which allows `import workerPath from 'hydrogen-view-sdk/main.js?url';` to still work even though we have `package.json` `exports` defined (`vite@2.9.1`).

And to includes fixes for https://github.com/vector-im/hydrogen-web/issues/722 which were added in https://github.com/vitejs/vite/issues/7822 ->https://github.com/vitejs/vite/pull/7827 (`vite@2.9.8`)